app/_hub/kong-inc/openid-connect/overview/_index.md

```diff
@@ -388,15 +388,18 @@ difficulties during this phase, please refer to the [Keycloak documentation](htt
    <br>
    <img src="/assets/images/products/plugins/openid-connect/keycloak-client-kong-auth.png">
    <br>
-2. Create another confidential client `service` with `client_secret_basic` authentication.
+2. Create a confidential client `service` with `client_secret_basic` authentication.
    For this client, Keycloak will auto-generate a secret similar to the following: `cf4c655a-0622-4ce6-a0de-d3353ef0b714`.
    Enable the client credentials grant for the client:
    <br><br>
    <img src="/assets/images/products/plugins/openid-connect/keycloak-client-service-settings.png">
    <br>
    <img src="/assets/images/products/plugins/openid-connect/keycloak-client-service-auth.png">
    <br>
-3. Create a verified user with the name: `john` and the non-temporary password: `doe` that can be used with the password grant:
+3. Optional: Create another confidential client `cert-bound` with settings similar to the `service` client created previously.
+   From the **Advanced** tab, enable the **OAuth 2.0 Mutual TLS Certificate Bound Access Tokens Enabled** toggle.
```

Comment:
I removed the screenshot here since we're trying to only use screenshots for dashboard type pages and I think the instructions explain what to enable.